consensus/storage_fault_monitor_test.go

```diff
@@ -0,0 +1,144 @@
+package consensus_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/abi"
+	"github.com/filecoin-project/go-filecoin/actor/builtin/miner"
+	"github.com/filecoin-project/go-filecoin/address"
+	. "github.com/filecoin-project/go-filecoin/consensus"
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+func TestStorageFaultSlasher_Slash(t *testing.T) {
+	tf.UnitTest(t)
+
+	ctx := context.Background()
+	signer, _ := types.NewMockSignersAndKeyInfo(1)
+
+	t.Run("When no miners, empty LateMiners", func(t *testing.T) {
```

Comment:
I'm having difficulty seeing what this test is testing. It looks like its just testing that Slash doesn't error when slashable miners is empty. If that's the case the test description should say that.